modules/tcia_dataset.ipynb

```diff
@@ -0,0 +1,579 @@
+{
```

Comment:
<p>based on comments from Justin...</p><ul><li>clarify that we can query the collections information using the REST API, e.g. collection name, abstract, image modalities, how many subjects/scans/images, DOI/URL of the Collection page that describes the data in detail https://wiki.cancerimagingarchive.net/display/Public/TCIA+Programmatic+Interface+REST+API+Guides</li><li>at the end of the demo, create a training example or refer the users to another notebook for deep learning segmentation</li><li>clarify how <code>download_len</code> works when the collections don't actually have sufficient volumes for a user-specified value</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/789/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>